### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
CC @vikivivi TIA

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -238,6 +238,7 @@ abtractor->abstractor
 abtracts->abstracts
 abudance->abundance
 abudances->abundances
+abundace->abundance
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:241]: Add `abundaces->abundances`?

**Review by DimitriPapadopoulos — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -3738,6 +3747,7 @@ authrorization->authorization
 authrors->authors
 autimatic->automatic
 autimatically->automatically
+autio->audio
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:3750]: In that case, we should make a script that can regex the contents of a morewords.com page and alter the root word with the typo for it. Then cherrypick the ones likely to be typos.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -2190,6 +2196,8 @@ apeture->aperture
 apetures->apertures
 apilogue->epilogue
 aplha->alpha
+aplicabile->applicable
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:2199]: both

File: codespell_lib/data/dictionary.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:3754]: @DimitriPapadopoulos Very inconvenient to copy paste and then clean up the results from morewords website. I'll skip this for now and add as an issue for us to follow-up on where we can also have a script that auto-checks and cleans up results for us...what say you?

[the remark this answers is not in the document]
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:3754]: Agreed, let'skip this. We do need assistance from a script at some point.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -9331,6 +9347,7 @@ cusstomization->customization
 cusstomize->customize
 cusstomized->customized
 cusstoms->customs
+custamizable->customizable
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:9357]: d349992

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -6546,8 +6559,11 @@ colaborations->collaborations
 colateral->collateral
 coldplg->coldplug
 colected->collected
+colecting->collecting
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:6567]: d349992

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
@luzpaz You need to rebase, this should also fix stalled CI jobs.

**Pull request merged**